src/main/scala/no/ndla/conceptapi/controller/DraftConceptController.scala

```diff
@@ -186,6 +186,54 @@ trait DraftConceptController {
 
       }
     }
+    
+    get(
+      "/subjects/",
+      operation(
+        apiOperation[List[String]]("getSubjects")
+          .summary("Returns a list of all subjects used in concepts")
+          .description("Returns a list of all subjects used in concepts")
+          .parameters(
+            asHeaderParam(correlationId)
+          )
+          .authorizations("oauth2")
+          .responseMessages(response400, response403, response404, response500))
+    ) {
+      readService.allSubjects() match {
```

Comment:
Bra utgangspunkt, men om du ser i readService så hentes allSubjects fra publishedConceptRepository. Du må gjøre tilsvarende men hente fra draftConceptRepository. Enten utvider du allSubjects med en parameter, eller du lager en egen metode med anna navn som gjør det samme.

Reply:
Skal ha fiksa det for denne og for allTagsFromConcept